libres/lib/enkf/enkf_main.cpp

```diff
@@ -341,12 +340,12 @@ static void enkf_main_node_std(const vector_type *ensemble,
     }
 }
 
-void enkf_main_inflate_node(enkf_main_type *enkf_main, enkf_fs_type *src_fs,
-                            enkf_fs_type *target_fs, int report_step,
+void enkf_main_inflate_node(const ensemble_config_type *ens_config,
+                            enkf_fs_type *src_fs, enkf_fs_type *target_fs,
+                            const int ens_size, int report_step,
                             const char *key, const enkf_node_type *min_std) {
-    int ens_size = enkf_main_get_ensemble_size(enkf_main);
     vector_type *ensemble = enkf_main_alloc_node_ensemble(
-        enkf_main, src_fs, key, report_step); // Was ANALYZED
+        ens_config, ens_size, src_fs, key, report_step); // Was ANALYZED
```

Comment:
Think we can get rid of the commend // Was ANALYZED

Reply:
Yeah - will do